manifests/dropin_file.pp

```diff
@@ -81,15 +81,15 @@
     File[$full_filename] ~> Service <| title == $unit or name == $unit |>
 
     if $daemon_reload {
-      Systemd::Daemon_reload[$unit] ~> Service <| title == $unit or name == $unit |>
+      Systemd::Daemon_reload[$unit] -> Service <| title == $unit or name == $unit |>
```

Comment:
If this isn't a notify anymore, should this be around line 77 instead? I think the ordering is still correct. Same for the other case.

Reply:
Thanks for taking a look at this @ekohl. The conditions stay the same. The ordering of the `Daemon_reload` in relation to the `Service` still only needs to happen when both `$notify_service` and `$daemon_reload` are true.